docs/how_tos/theming.md

```diff
@@ -0,0 +1,204 @@
+# Theming support with `@openedx/paragon` and `@openedx/brand-openedx`
+
+## Overview
+
+This document serves as a guide to using `@openedx/frontend-platform` to support MFE theming with Paragon using theme CSS loaded externally (e.g., from a CDN).
+
+To do this, configured URLs pointing to relevant CSS files from `@openedx/paragon` and (optionally) `@openedx/brand-openedx` are loaded and injected to the HTML document at runtime. This differs than the consuming application importing the styles from `@openedx/paragon` and `@openedx/brand-openedx` directly, which includes these styles in the application's production assets.
+
+By serving CSS loaded externally, consuming applications of Paragon no longer need to be responsible for compiling the theme SCSS to CSS themselves and instead use a pre-compiled CSS file. In doing so, this allows making changes to the Paragon theme without needing to necessarily re-build and re-deploy all consuming applications.
+
+### Dark mode and theme variant preferences
+
+`@openedx/frontend-platform` supports both `light` (required) and `dark` (optional) theme variants. The choice of which theme variant should be applied on page load is based on the following preference cascade:
+
+1. **Get theme preference from localStorage.** Supports persisting and loading the user's preference for their selected theme variant, until cleared.
+1. **Detect user system settings.** Rely on the `prefers-color-scheme` media query to detect if the user's system indicates a preference for dark mode. If so, use the default dark theme variant, if one is configured.
```

Comment:
The hook will fallback in the installed themes files if the URL is not provided or is not working. 

If you define only the CSS inside SCSS file you can use the `prefers-color-scheme`  to support dark/light modes.

Reply:
Thank you! It would be great to explicitly indicate exactly how it is possible to use different imports for a light/dark theme in the documentation. I plan to do this in the migration documentation and for Paragon.

For example:
```
@use "@openedx/paragon/dist/core.min.css" as paragonCore;
@import url('@openedx/paragon/dist/light.min.css') (prefers-color-scheme: light);
@import url('@my-brand/dark.min.css') (prefers-color-scheme: dark);
```

Reply:
100% agree that the information in your example should be part of the Paragon and migration documents.

This is a different approach, the explanation of how to use variants is below, in the title  **Basic theme URL configuration**.

 I will improve one of the examples to use dark mode, to make it more explicit.